07-performance.Rmd

```diff
@@ -4,16 +4,24 @@ knit: "bookdown::preview_chapter"
 
 # Efficient optimization {#performance}
 
-[Donald Knuth](https://en.wikiquote.org/wiki/Donald_Knuth) is a famous American
-computer scientist who developed a number of the key algorithms that we use today.
-On the subject of optimisation he give this advice.
+[Donald Knuth](https://en.wikiquote.org/wiki/Donald_Knuth) is a legendary American
+computer scientist who developed a number of the key algorithms that we use today (see for 
+example `?Random`).
```

Comment:
Great to direct people to R help for further info - just took a look and it's interesting that his name pops up there.